components/eam/cime_config/config_component.xml

```diff
@@ -51,7 +51,6 @@
 
       <value compset=""                        >-mach $MACH</value>
       <value compset="_EAM"                    >-phys default</value>
-      <value compset="_EAM%CMIP6_"             >&eamv3_phys_defaults; &eamv3_chem_defaults;</value>
```

Comment:
This will cause problems for MMF coupled compsets (currently only exist in a branch) that use "1850SOI_EAM%CMIP6-MMF1_" in the compset long name. I've been meaning to make a PR to add these compsets, but keep forgetting to do it. Can we just add another line below like we did for the "pct" and "4xCO2" modifications?

Reply:
@whannah1 , Will the compset be using eamv3 physics and chemistry? In that case, it is covered by line 56 (matching negative SSP lookaround) Otherwise, will need an extra line to distinguish, like the current fix for SSP. Or if set to not using vb3, need to add a negative lookaround of MMF1 to line 56.

Reply:
After chatting with Wuyin offline I think this is ok.

Reply:
@wlin7 to answer your question - the MMF-WCYCL compsets will not use the v3 physics or chem options, so we'll need to address that later. 

As a side note - it would be really nice to be able to specify the compsets use something like "_MMF%CMIP6_", but I recall trying to do that years ago and running into a lot of problems.

Reply:
The string before the "%" is the model which is EAM.   After the percent is where you specify configure/namelist options so "EAM%MMF_CMIP6" is what you should aim to get working.